pkg/operator/ceph/operator.go

```diff
@@ -61,6 +62,13 @@ var (
 	EnableFlexDriver = true
 	// Whether to enable the daemon for device discovery. If true, the rook-ceph-discover daemonset will be started.
 	EnableDiscoveryDaemon = true
+
+	// ImmediateRetryResult Return this for a immediate retry of the reconciliation loop with the same request object.
+	ImmediateRetryResult = reconcile.Result{Requeue: true}
```

Comment:
I'm still not understanding the scenarios for immediately re-queueing. For example, if there is an error we should wait some number of seconds before retrying.

Reply:
Make sense but how long?

Reply:
offline Seb confirmed it backs off, so it's all good